notation.go

```diff
@@ -41,6 +41,9 @@ import (
 )
 
 var errDoneVerification = errors.New("done verification")
+
+const errReferrersTagSchemaDelete = "failed to delete dangling referrers index"
```

Comment:
I'm curious if there's a corresponding error type defined in `oras-go`? If yes, we should use that one.

Reply:
Updated.